BrowserKit/Tests/WebEngineTests/Mock/MockFullscreenDelegate.swift

```diff
@@ -0,0 +1,21 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+@testable import WebEngine
+
+class MockFullscreenDelegate: FullscreenDelegate {
```

Comment:
Nit: `final`.

Reply:
If it's ok I will make the change in the next PR where I will add support for fullscreen to the app